_opensearch/query-dsl/full-text.md

```diff
@@ -392,44 +416,33 @@ GET _search
   }
 }
 ```
+-->
+## Advanced filter options
 
-## Convert text with analyzers
-
-To convert unstructured text into the format that you need, you can use the text analyzers. To learn more about how to convert unstructured text into structured text that is optimized for search, see [Text analyzers]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/text-analyzers).
-
-<!-- TO do: each of the options needs its own section with an example. Convert table to individual sections, and then give a streamlined list with valid values. -->
-
-## Optional query fields
-
-You can filter your query results by using some of the optional query fields, such as wildcards, analyzers, fuzzy query fields, and synonyms.
+You can filter your query results by using some of the optional query fields, such as wildcards, fuzzy query fields, and synonyms.
```

Comment:
Just a note: You **can** use analyzers as optional query fields (like in the match phrase query above).

Reply:
added link to how to use analyzers section, as well.